src/main/java/kobe/Commands.java

```diff
@@ -0,0 +1,37 @@
+package kobe;
+
+import java.lang.System;
+
+public class Commands {
+    public static String ind = "    ";
+    public static String line = ind + "____________________________________________________________\n" + ind;
```

Comment:
Also, i do not think that Command class is something that deals with Ui process, cause for this class you are using indentation and line as its attributes, which sounds not like a Command. Just my opinion.